docs/pages/User/GettingStarted/install.md

```diff
@@ -55,3 +55,19 @@ Therefore you may need to repeat the above procedure on another cluster in order
 
 Once you have two clusters ready, you can start the peering procedure, which is presented in the [next step](../peer).
 
+## Uninstall
```

Comment:
@palexster I would suggest to create a new, dedicated page for the uninstall. Your opinion?

Reply:
Totally agree, otherwise it would break the tutorial "flow".